README.md

```diff
@@ -1388,23 +1388,27 @@ The default `query` (`spacy.CallLangChain.v1`) executes the prompts by running `
 
 Interacting with LLMs, either through an external API or a local instance, is costly.
 Since developing an NLP pipeline generally means a lot of exploration and prototyping,
-`spacy-llm` implements a built-in cache to avoid reprocessing the same documents at each run.
+`spacy-llm` implements a built-in cache to avoid reprocessing the same documents at each run
+that keeps batches of documents stored on disk.
 
 Example config block:
 
 ```ini
 [components.llm.cache]
-@llm_misc = "spacy.BatchCache.v1",
+@llm_misc = "spacy.BatchCache.v1"
 path = "path/to/cache"
 batch_size = 64
 max_batches_in_mem = 4
 ```
 
-| Argument             | Type                         | Default | Description                                          |
-| -------------------- | ---------------------------- | ------- | ---------------------------------------------------- |
-| `path`               | `Optional[Union[str, Path]]` | `None`  | Cache directory. If `None`, no caching is performed. |
-| `batch_size`         | `int`                        | 64      | Number of docs in one batch (file).                  |
-| `max_batches_in_mem` | `int`                        | 4       | Max. number of batches to hold in memory.            |
+| Argument             | Type                         | Default | Description                                                                                                               |
+| -------------------- | ---------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `path`               | `Optional[Union[str, Path]]` | `None`  | Cache directory. If `None`, no caching is performed, and this component will act as a NoOp.                               |
+| `batch_size`         | `int`                        | 64      | Number of docs in one batch (file). Once a batch is full, it will be peristed to disk.                                    |
```

Comment:
```suggestion
| `batch_size`         | `int`                        | 64      | Number of docs in one batch (file). Once a batch is full, it will be persisted to disk.                                   |
```